skype/skype-ps/skype/New-CsExternalUserCommunicationPolicy.md

```diff
@@ -144,21 +143,15 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Tenant
+Globally unique identifier (GUID) of the tenant account whose external user communication policy are being created. For example:
+
```

Comment:
This is repeated below.  Delet this instance

Reply:
Done!